in/landuse/landuseclass.def

```diff
@@ -0,0 +1,9 @@
+1        Beets                     in/landuse/Beets.PAR
+2        Coniferous Forest         in/landuse/Coniferous_forest.PAR
+3        Corn                      in/landuse/Corn.PAR  
+4        Crops & Vegetables        in/landuse/Crops_vegetables.PAR
+5        Deciduous_forest          in/landuse/Deciduous_forest.PAR
```

Comment:
Classname `Deciduous_forest` should be `Deciduous Forest` to be consistent with Line 2.